### ISSUE TITLE
Fetching product data is slow

### ISSUE THREAD

**Issue by axhx** [Reshav-Paul/grodudes_app#3]
1. While opening the application sometimes the application shows all the products and sometimes it gets loads for more times.

2. While selecting some category(example:fruits) it keeps on loading and loading and doesn't display the items. when we close and restart the application it shows products properly.

How to resolve this issue and please guide and help me to resolve this issue.

**Comment by Reshav-Paul** [comment]
Have you modified the code to fetch all products under a category using the hack I shared?

If yes, then it will be slow depending on how much data you are fetching i.e. the number of items in that category. The solution is to implement a dynamic loading list as is implemented in Orders.dart.

If you have not made any modifications to the code, the server itself might be slow.

Have you implemented the looping solution I shared?
How many products are you fetching?
